server/routes/cards.js

```diff
@@ -38,14 +38,54 @@ router
   })
   .post((req, res) => {
     // check if word exists
-    new Word('english_word', req.body.english_word)
+    console.log('reached check if card exists')
+    const word = req.body.english_word.toLowerCase();
+    new Word('english_word', word)
       .fetch()
       .then((wordResult) => {
+        console.log('fetched word to check if exists');
         if (!wordResult) {
+          console.log('word does not exist; reached created word')
+          console.log('word', word)
           // creates new row in Word table
-          return new Word().save({ english_word: req.body.english_word }).then((result) => {
-            return res.json(result);
-          });
+          new Word()
```

Comment:
if you felt so-inclined, you could replace this nested promise with a chained one.
Something like:

if (wordResult) {
  // if word already there then just go to next promise
  return;
} else {
  return new Word().save({ --- })
}}.then((result) => { ....

additionally you'd have to re-arrange some of your opening/closing parens... you can decide whether it's worth the effort